<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LA-calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>221</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_B">
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit_B">
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_B">
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_transpose_B">
         <property name="text">
          <string>Transpose</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_rank_B">
         <property name="text">
          <string>Rank</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_det_B">
         <property name="text">
          <string>Determinant</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_inverse_B">
         <property name="text">
          <string>Inverse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>221</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_A">
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit_A">
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_A">
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_transpose_A">
         <property name="text">
          <string>Transpose</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_inverse_A">
         <property name="text">
          <string>Inverse</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_rank_A">
         <property name="text">
          <string>Rank</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_det_A">
         <property name="text">
          <string>Determinant</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>101</width>
      <height>124</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_operators">
     <item>
      <widget class="QPushButton" name="pushButton_switch">
       <property name="text">
        <string>switch</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_plus">
       <property name="text">
        <string>A + B</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_minus">
       <property name="text">
        <string>A - B</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_mult">
       <property name="text">
        <string>A * B</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>310</y>
      <width>221</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextBrowser" name="textBrowser_result"/>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_to_A">
         <property name="text">
          <string>Insert into A</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_copy_tex">
         <property name="text">
          <string>Copy LaTeX</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_copy">
         <property name="text">
          <string>Copy array</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_to_B">
         <property name="text">
          <string>Insert into B</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_tex_A">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>221</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Paste LaTeX code</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tex_B">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>350</y>
      <width>221</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Paste LaTeX code</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>280</y>
      <width>221</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_power_B">
       <property name="text">
        <string>To the power of</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_multiply_B">
       <property name="text">
        <string>Multiply by</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinBox_power_B"/>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBox_multiply_B">
       <property name="minimum">
        <number>-1000</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>221</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_power_A">
       <property name="text">
        <string>To the power of</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinBox_power_A"/>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_multiply_A">
       <property name="text">
        <string>Multiply by</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBox_multiply_A">
       <property name="minimum">
        <number>-1000</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>875</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCalculator">
    <property name="title">
     <string>Calculator</string>
    </property>
    <widget class="QMenu" name="menuAlgorithms">
     <property name="title">
      <string>Algorithms</string>
     </property>
     <addaction name="actionGaussian_Elimination"/>
     <addaction name="separator"/>
     <addaction name="actionSymmetric_Gauss"/>
    </widget>
    <addaction name="actionSystem_of_Equations"/>
    <addaction name="menuAlgorithms"/>
   </widget>
   <addaction name="menuCalculator"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSystem_of_Equations">
   <property name="text">
    <string>System of Equations</string>
   </property>
  </action>
  <action name="actionGaussian_Elimination">
   <property name="text">
    <string>Gaussian Elimination</string>
   </property>
  </action>
  <action name="actionExp_Gaussian_Elimination">
   <property name="text">
    <string>Exp Gaussian Elimination</string>
   </property>
  </action>
  <action name="actionSymmetric_Gauss">
   <property name="text">
    <string>Symmetric Gauss</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
